CHANGELOG.md

```diff
@@ -30,6 +30,197 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v9.0.0](https://github.com/shopsys/shopsys/compare/v7.3.3...v9.0.0) - 2020-05-18
```

Comment:
Do not forget to update date before merging

Reply:
i will try it :smiley: